coralproject/cay
# Added a hook to React router

## What does this PR do?
- Adds a warning when trying to exit a form that has changes

## How do I test this PR?
- Go to **Create form**
- Click on other link ("View forms", perhaps?) -> you should navigate with no problem
- Go to **Create form** again
- Change something (title, add fields, etc)
- Click on another link to exit form creation -> a dialog should prevent you

Note:

This PR adds a simple hook on React-Router's API which works well and gives us a native browser warning as many apps do:

<img width="588" alt="captura de pantalla 2016-07-24 a las 17 58 56" src="https://cloud.githubusercontent.com/assets/340766/17086459/06632ce4-51c9-11e6-9119-72718ee710f7.png">

In order to reflect the design on #442 , a slightly more complex solution should be coded. 


@coralproject/frontend
